strax/processing/peak_merging.py

```diff
@@ -160,6 +174,10 @@ def add_lone_hits(peaks, lone_hits, to_pe):
         p['area'] += lh_area
         p['area_per_channel'][lh_i['channel']] += lh_area
 
-        # Add lone hit as delta pulse to waveform:
+        # Add lone hit as delta pulse to waveforms:
         index = (p['time'] - lh_i['time'])//p['dt']
```

Comment:
This line is wrong. Effectively it will lead to a mirrored relative timestamp inside a merged S2 peak. Let's do one thing each time and fix this in another PR. It should be 
```index = ( lh_i['time']-p['time'])//p['dt']```